### ISSUE TITLE
sssctl: obtain talloc report

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#3144]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/3149
- Created at 2016-08-22 12:45:04 by [pbrezina](https://pagure.io/user/pbrezina)
- Closed at 2020-03-24 14:16:28 as wontfix
- Assigned to nobody
---

We want to be able to obtain talloc report through sssctl.

### Comments

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-08-29 11:45:31](https://pagure.io/SSSD/sssd/issue/3149#243235)**

Deferring until we need this urgently (at the moment, a gdb one-liner can be used as a workaround)

milestone: NEEDS_TRIAGE => SSSD Deferred

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2016-08-31 17:15:44](https://pagure.io/SSSD/sssd/issue/3149#243236)**

Fields changed

rhbz:  => todo

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2017-02-24 15:08:21](https://pagure.io/SSSD/sssd/issue/3149#243237)**

**Metadata Update from @pbrezina**:
- Issue marked as depending on: #3148
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:16:27](https://pagure.io/SSSD/sssd/issue/3149#636780)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [pbrezina](https://pagure.io/user/pbrezina) at [2020-03-24 14:16:28](https://pagure.io/SSSD/sssd/issue/3149#636781)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
